libraries/GCS_MAVLink/GCS.h

```diff
@@ -448,6 +450,9 @@ class GCS_MAVLINK
 
 private:
 
+    // reduce effect of vehicle Yaw and gimbal pitch with increasing camera zoom
+    MAV_RESULT handle_command_do_scale_with_zoom(const mavlink_command_long_t &packet);
```

Comment:
This should be declared in the same places where other `handle_mav_cmd...` methods are declared.  Alternatively, since this is only currently used in copter, you can declare/implement this in the `GCS_Copter` derived class

Reply:
do_scale_with_zoom should probably stay in the common class so it can be used with both copter and fixed wing gimbals